src/filters/Applications.ts

```diff
@@ -0,0 +1,13 @@
+export function applicationHasDeploymentFilter ($filter: any) {
```

Comment:
lets not recreate these filters in catalog with slightly different names, I don't want this in multiple repos, its going to be a pain to track down if it ever changes. Move the filters up to common if needed.